### DISCUSSION TITLE
how to sync Sharepoint Library (not one drive shared) with one drive client on Zorin OS 16 (Ubuntu 20.04)

### DISCUSSION THREAD

**Discussion by apanse418** [abraunegg/onedrive#2422] [Q&A] [answered]
I thought that https://github.com/abraunegg/onedrive/blob/master/docs/SharePoint-Shared-Libraries.md this is going to help me setup a sync of my share point library a standard document library that i have created on share point. Please note this is not a shared library in one drive and I think this is where I went wrong following the above linked document. It seem the procedure is valid only for shared libraries in one drive.
The share point site is not a personal one but is on a business account and I am the owner of it so I have credentials and admin access on the site and folders.
Can someone guide me how do I sync it to my Ubuntu Linux desktop ( I am using Zorin OS)
```
$ uname -a
5.15.0-75-generic #82~20.04.1-Ubuntu SMP Wed Jun 7 19:37:37 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux

lsb_release -a
No LSB modules are available.
Distributor ID:	Zorin
Description:	Zorin OS 16.2
Release:	16
Codename:	focal
```

I am able to retrieve the site name using -get-O365-drive-id command it properly shows me my site name, library names which is "Documents" drive_id and Library URL which is a shared documents URL.

Any help setting this one up is greatly appreciated and I apologies if I am repeating this question however I could not find one all of them are related to shared library in one drive and not the actual share point library, may be I didn't look at right place ?

**Comment by abraunegg** [comment, marked as answer]
@apanse418 
A SharePoint Library is a Shared SharePoint Library - as that is how it is configured by OneDrive | Office365 for your business account. You might be the 'only' person using it or has access to it - but it is the correct nomenclature.

So following the document [SharePoint-Shared-Libraries.md](https://github.com/abraunegg/onedrive/blob/master/docs/SharePoint-Shared-Libraries.md) is the correct process to set this up to get it working.

**Comment by apanse418** [comment]
@abraunegg thank you for replying to it so quickly appreciate it. It seems I was using the wrong config file as I was typing the commands as it is from your guide. But today morning I sat with a fresh mind and it struck me that when I was running config file display it was showing me one drive sync_dir instead my share point folder I created locally to sync. So I resolved it. Thanks a lot for all your efforts to keep this updated and supported. appreciate it.
